website/docs/docs/cloud/manage-access/enterprise-permissions.md

```diff
@@ -4,14 +4,9 @@ id: "enterprise-permissions"
 description: "Permission sets for Enterprise plans."
 ---
 
-:::info Enterprise Feature
+import SetUpPages from '/snippets/_available-enterprise-only.md';
 
-This guide describes a feature of the dbt Cloud Enterprise plan.
-If you're interested in learning more about an Enterprise plan, contact us at sales@getdbt.com.
-
-:::
-
-## Overview
+<SetUpPages features={'/snippets/_available-enterprise-only.md'}/>
```

Comment:
Hey @runleonarun & @mirnawong1, just want to call out one thing here (left the same note in the [documentation pr](https://github.com/dbt-labs/docs.getdbt.com/pull/3624#pullrequestreview-1514768046)).

When using the partials, the file path does not have to be included when using the partial component. It can be written as:
```markdown
import SetUpPages from '/snippets/_available-enterprise-only.md';

<SetUpPages />
```

Props (the `features={...` part on the component) only need to be added if you want to pass in data from the component into the partial. This is useful for using the same partial component on multiple docs pages and displaying different values for each.

For example, if we wanted to use this partial on multiple pages and pass in a different 'feature' for each docs page, we can write it as:
```markdown
import SetUpPages from '/snippets/_available-enterprise-only.md';

<SetUpPages feature='A really cool feature' />
```

Then within the `/snippets/_available-enterprise-only.md` file, we can display that feature prop with:
```markdown
This feature: {props.feature}

other content etc...
```

That would allow us to use the same component on a different docs page, where we can show a separate feature, for example:
```markdown
import SetUpPages from '/snippets/_available-enterprise-only.md';

<SetUpPages feature='Another awesome feature' />
```